### DISCUSSION TITLE
What determines floppy read data?

### DISCUSSION THREAD

**Discussion by Elektraglide** [keirf/greaseweazle#300] [Q&A]
I have a single original 360k floppy disk for a Tektronix 4404 workstation.  It boots just fine.
I've used GW to make an IMG  file of it - and a sniff test of the content looks fine.
I have written a new floppy with this IMG.  (Also read same floppy back and _cmp srcfile newfile_ reports identical).

However, this new floppy fails to boot.

Q: Surely if they are bit-identical, it should boot, no?  Unless the difference lies in the Tek4404 reading the disk rather than GW and getting a different result.  Is that possible?  

Any ideas?  Should I use _gw erase_ first?

**Comment by keirf** [comment]
It could be that the Greaseweazle format analyser is incomplete for this disk format and is not getting the disk format correct enough to be accepted by the Tektronix. That's probably unlikely though.

I would say it's probably more likely that the Tektronix drive is fussier and doesn't like the disk written by the Greaseweazle. It could be that one drive or the other is a little bit out of alignment for example. 

Erasing the disk before you write it is a good idea. You can use gw erase with its high frequency option. An even better erasure method is to wipe a strong magnet across the disk jacket. A small neodymium magnent such as you find in old hard drives is good.

**Comment by Elektraglide** [comment]
OK, found a ROM utility on Tek4404 to write floppy tracks.  If I create a .SCP of tracks0 & 1,  will it give insight into what format the native diskcontroller is writing (and therefore expecting)?

[read_tek4404.SCP.zip](https://github.com/keirf/greaseweazle/files/11254179/read_tek4404.SCP.zip)

**Comment by keirf** [comment]
You need to do a raw SCP read. I think you specified `--format` on the command line which regenerates the track data according to the specified format, before generating the SCP.

**Comment by Elektraglide** [comment]
Doh. You are wise.  Will give it another go when I get back home.
